Project.toml

```diff
@@ -12,8 +12,8 @@ Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
-EzXML = "0.8, 0.9, 1"
-Mocking = "0.5.6, 0.6, 0.7"
+EzXML = "0.9.1, 1"
```

Comment:
Is there a reason for not using `0.9`?

Reply:
`0.9.1` was the first revision with a Project.toml. I mentioned this in the comment message: https://github.com/JuliaTime/TimeZones.jl/pull/242/commits/eb0263b1005b03a488105231bb7bac6dc6c12211

Reply:
Proper commit messages 😍 , I'll have to get back into the habit of reading them!